docs/content/latest/benchmark/tpcc-ysql/1000-wh/tpcc-ysql.md

```diff
@@ -10,36 +12,55 @@ $ ./tpccbenchmark --create=true --load=true \
   --loaderthreads 48
 ```
 
+<table>
+  <tbody>
+    <tr>
+      <td>Cluster</td>
+      <td>3 nodes of type `c5d.4xlarge`</td>
+    </tr>
+    <tr>
+      <td>Loader threads</td>
+      <td>48</td>
+    </tr>
+    <tr>
+      <td>Loading Time</td>
+      <td>~10 hours</td>
+    </tr>
+    <tr>
+      <td>Data Set Size</td>
+      <td>~420 GB</td>
```

Comment:
Are these 3-way replicated sizes? Should these numbers be 1/3rd of what we're stating here?

Reply:
Not really sure. 
Whether we should put physical or logical sizes here.
Our cluster gives out these details. So shouldn't it be better to retain these values.

And Maybe specify that this is physical usage and not logical.